release/RELEASE.md

```diff
@@ -1,149 +1,157 @@
+time.
+
+> **When you create the release branch in the [daml] repo, make a PR to `main`
+> bumping [`NIGHTLY_PREFIX`] accordingly.**
+
+The steps to create a release are:
+
+1. In the [daml] repo, edit the [`LATEST`] file. The format of that file is a
+   commit sha of the code you want to release (at this point typically the tip
+   of the release branch), the version number you want to attribute to the
+   build, and the words `SPLIT_RELEASE` (for historical reason dating to
+   pre-2.0 days).  For a release candidate, one can generate a snapshot version
+   string using the `release.sh` script. For example, to create a PR for a
+   2.0.0 release candidate:
 
-> **IMPORTANT**: If the release fails, please delete it from the [releases page]
-> and write how it failed on the PR.
-
-First, you need to know which type of release you are making. At this point in
-time, these are the options, in rough order of likelihood/frequency:
-
-1. The weekly snapshot.
-2. A release candidate for a 2.x release.
-3. A stable 2.x release.
-4. A release candidate for a 1.x release.
-5. A stable 1.x release.
-
-If you don't know which of these fit your current case, or you think it
-doesn't fit in any, please reach out on `#team-daml` on Slack, preferably
-mentioning `@gary`.
-
-## The weekly snapshot
-
-> Note: You should not have to make any change to this repo.
-
-1. On Wednesday morning, a cron should create a release PR on the [assembly]
-   repo, mentioning you. Please reach out to `@gary` on Slack if that's
-   missing.
-
-2. Merge the PR and wait for the corresponding `main` build to finish.
-
-3. Go to the [Testing](#testing) section of this file.
-
-## 2.x release candidate
-
-In a perfect world, there is no need for a separate RC: the latest weekly
-snapshot can work. In the world we live in though, we frequently want to add a
-few things, or patch previous minor releases.
-
-In those cases, we create `release/` branches (e.g. `release/2.0.x`). Those are
-special branches, protected by GitHub rules and treated specially by CI.
+   ```
+   $ git fetch
+   $ ./release.sh snapshot origin/release/2.0.x 2.0.0
+   ```
 
-When making a release candidate, you generally want to pick the tip of one of
-those release branches. The release itself is always triggered from `main`.
+   You should put that line in the file in order to preserve semver ordering,
+   and overwrite any existing snapshot with the same prefix.
 
-The process is similar to the weekly snapshot, except that both the [daml] and
-[canton] bundles have to be manually triggered. Specifically:
+   Stable releases should use the same code as the last RC, so instead of
+   adding a new line you should simply remove the `-snapshot.*` part of the
+   version string on the appropriate line of the [`LATEST`] file.
 
+2. Make a PR **targeting the `main` branch** with just that one line added,
+   touching no other file. Add the `Standard-Change` label to that PR.
```

Comment:
Is there a set of default reviewers ?

Reply:
Not really, no. For minor releases I tend to try and get "someone important" (i.e. someone from the "Product Managers and Engineering Directors" list in the release document) to sign off, but for obvious reasons they tend to be quite busy. Historically we've tried to get at least one team-lead-level approval (considering "opening the PR" as an approval too).